ethcore/src/miner/stratum.rs

```diff
@@ -0,0 +1,233 @@
+// Copyright 2015, 2016 Ethcore (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Client-side stratum job dispatcher and mining notifier handler
+
+use ethcore_stratum::{
+	JobDispatcher, PushWorkHandler,
+	Stratum as EthcoreStratum, Error as StratumServiceError,
```

Comment:
`EthcoreStratum` is a really uninformative name here. Seems to be a stratum service? Maybe import it as `StratumService`?